pom.xml

```diff
@@ -3,8 +3,8 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.ryantenney.metrics</groupId>
 	<artifactId>metrics-spring</artifactId>
-	<version>3.1.3-SNAPSHOT</version>
-	<packaging>bundle</packaging>
+	<version>3.9.9-SNAPSHOT</version>
+	<packaging>jar</packaging>
```

Comment:
I'm not too familiar with the osgi stuff, and it was giving odd errors, but can restore

Reply:
The bundle packaging didn't seem to work, haven't investigated